app/authentication/api_keys.py

```diff
@@ -35,22 +37,22 @@ async def __call__(
 
 
 async def get_api_key(
-    api_key_query: Tuple[Optional[str], Optional[str], Optional[str]] = Security(
-        APIKeyOriginQuery(name=API_KEY_NAME, auto_error=False)
-    ),
+    # api_key_query: Tuple[Optional[str], Optional[str], Optional[str]] = Security(
+    #     APIKeyOriginQuery(name=API_KEY_NAME, auto_error=False)
+    # ),
```

Comment:
API Gateway doesn't support api key in query string out of the box. If we're committed to supporting it, can dig into what API Gateway calls custom authorizers.

Reply:
Confirmed that all clients are passing api key in header